src/cgame/cg_main.c

```diff
@@ -378,280 +379,289 @@ vmCvar_t cg_crosshairHealth;
 
 vmCvar_t cg_commandMapTime;
 
+// vmCvar_t Descriptions {{{
+#define cg_autoCmd_description \
+		"Allows setting a comand (or commands, ';'-seperated), that will automatically be executed when you first join a match or when the map restarts.\n" \
+		"\n" \
+		"Example:\n" \
+		"  /cg_autoCmd \"team r 2 23 3;say The sanctity of this place has been fouled.\""
+// {{{
+
```

Comment:
Instead of this, I would add a struct + array, like `cvarTable_t`, which holds a pointer to a `vmCvar_t` + description for the cvar. You could then iterate this array in `CG_RegisterCvars`, where you currently check if the description is set, and register all the descriptions for cvars that way. I understand this is probably a temp solution that you've provided here, but this would be my suggestion for a more permanent solution, rather than adding hundreds of macros here, or writing out the descriptions to the cvartable itself.